elife/mysql-client.sls

```diff
@@ -4,5 +4,4 @@ mysql-package-list:
     pkg.installed:
         - pkgs:
             - mysql-client
-            - python-mysqldb # py2, to be removed
```

Comment:
turns out we need to keep this until we're running on upgraded Salt. Makes sense